app/src/androidTest/java/com/google/android/apps/flexbox/test/MainActivityTest.java

```diff
@@ -16,19 +16,6 @@
 
 package com.google.android.apps.flexbox.test;
 
-import static android.support.test.espresso.Espresso.onView;
```

Comment:
Please restore the import order here.
I've updated the CONTRIBUTING.md to follow the official AndroidStyle code style.
Sorry wasn't updated before this PR.